src/test/groovy/pl/confitura/jelatyna/user/admin/AdminControllerSpec.groovy

```diff
@@ -83,6 +83,6 @@ class AdminControllerSpec extends AbstractControllerSpec {
 
     @Override
     def getControllerUnderTest() {
-        return new AdminController(repository, generator, emailSender)
+        return new AdminController(repository, new AdminService(repository, generator, emailSender))
```

Comment:
jesli serwis dostaje repo, to controller juz go nie powinien dostawac. Albo idziemy rpzez serwis albo nie.